DEV_3/CalculationMethod.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Text;
+using System.Collections;
+
+namespace DEV_3
+{
+        public class CalculationMethod
+        {
+            string letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+            /// <summary>
+            /// Translates from the decimal number system to the number system with the base of number system
+            /// </summary>
+            /// <param name="number">The number that we convert</param>
+            /// <param name="sys">Number system, into which we convert</param>
+            /// <returns></returns>
+            public string ToNumberSystem(string number, string sys)
+            {
+                string newNum = String.Empty;

```

Comment:
Are you sure that number will be converted? Use Parse and catch exceptions

Reply:
But I'm doing an argument check in main.